data/profiles/solarstorm2012.profile

```diff
@@ -0,0 +1,124 @@
+{
+  "actions": [
+    {
+      "documentation": "Reset button. Sets time to start at start of dataset. No loop",
+      "gui_path": "2012July",
+      "identifier": "2012July.Reset-loop",
```

Comment:
I believe we are (and if we don't, we should) stick to all lower case for the identifiers to reduce the cognitive load to remember whether something was lower case or upper case.  By making everything lower case, it'll be easy to remember

Reply:
Same argument for only using an underscore rather than a dot;  so:
`2021july.reset_loop` instead just for consistency with how we write the identifies of our scene graph nodes